docs/data-dictionary/models/saneringskrediet.yml

```diff
@@ -10,8 +10,10 @@ properties:
     $ref: "vorderingen.yml"
   betalingen:
     $ref: "betalingen.yml"
-  depot:
```

Comment:
@Frank-SBN wat is depot en moet die nog bestaan?

Reply:
@Frank-SBN Volgens mij is dit veld het bedrag wat gevorderd is maar wat nog niet is vervallen. Het saldo van vorderingen, betalingen en depot zou 0 moeten zijn wanneer er geen achter of voorstanden zijn. Is dit correct?